src/test/kotlin/org/treeWare/mySql/operator/MySqlTestContainer.kt

```diff
@@ -0,0 +1,13 @@
+package org.treeWare.mySql.operator
```

Comment:
Move this file to `testFixtures` so that it can be used in the tests of other projects:

`src/testFixtures/kotlin/org/treeWare/mySql/test/`